src/virus_clade_utils/util/sequence.py

```diff
@@ -54,6 +62,74 @@ def get_covid_genome_data(released_since_date: str, base_url: str, filename: str
     logger.info("NCBI API call completed", elapsed=elapsed)
 
 
+def download_covid_genome_metadata(url: str, data_path: Path) -> Path:
+    """Download the latest GenBank genome metadata data from Nextstrain."""
+
+    session = get_session()
+    filename = data_path / Path(url).name
+
+    start = time.perf_counter()
+    with session.get(url, stream=True) as result:
+        result.raise_for_status()
+        with open(filename, "wb") as f:
+            for chunk in result.iter_content(chunk_size=None):
```

Comment:
Reading the downloaded metadata in chunks isn't strictly necessary at the moment, but it's a hedge against an eventual file size that's too large for memory.